pkg/provider/resource_cluster.go

```diff
@@ -88,77 +91,81 @@ func (c *ClusterResource) Schema() *schema.Resource {
 				Required:    true,
 			},
 			"created_at": {
-				Description: "Cluster Creation Time",
+				Description: "Cluster Creation Time.",
 				Type:        schema.TypeString,
 				Computed:    true,
 			},
 			"deleted_at": {
-				Description: "Cluster Deletion Time",
+				Description: "Cluster Deletion Time.",
 				Type:        schema.TypeString,
 				Computed:    true,
 			},
 			"expired_at": {
-				Description: "Cluster Expiry Time",
+				Description: "Cluster Expiry Time.",
 				Type:        schema.TypeString,
 				Computed:    true,
 			},
 			"first_recoverability_point_at": {
-				Description: "Earliest Backup recover time",
+				Description: "Earliest Backup Recover Time.",
 				Type:        schema.TypeString,
 				Computed:    true,
 			},
 			"instance_type": {
-				Description: "Instance Type",
+				// TODO: Add validation for the instance_type
+				Description: "Instance Type. e.g. \"azure:Standard_D2s_v3\", \"aws:c5.large\".",
 				Type:        schema.TypeString,
 				Required:    true,
 			},
 			"cluster_id": {
-				Description: "cluster ID",
+				Description: "Cluster ID.",
 				Type:        schema.TypeString,
 				Computed:    true,
 			},
 			"connection_uri": {
-				Description: "cluster connection uri",
+				Description: "Cluster connection uri.",
```

Comment:
URI

Reply:
Done, because of the rebase and force-push, it looks Outdated.